README.md

```diff
@@ -144,14 +144,25 @@ column for example. Your choice depends on what your database supports. For
 example, create a migration like this:
 
 
+#### For PostgreSQL
```

Comment:
This needs to be read generally, like `For databases with ActiveRecord json data type support(e.g. PostgreSQL, MySQL)`